### PR TITLE
Tf 01 009 billing and payroll

### PR DESCRIPTION
**Pull request by Rasheduluctit**
Created The Billing and Payroll page...
[screen-capture (1).webm](https://github.com/UCT-IT/edume/assets/149924448/0ce70b54-0107-4ef7-946a-8992ca746f02)
localhost_3000_pages_billingAndPayroll

### CODE REVIEW

File: src/app/pages/billingAndPayroll/customizableInvoices/RightComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import React from "react";
+import billingImg3 from "../../../../../public/featuresPages/billingAndPayroll/billing-image-3.png";
+
+const RightComponent = () => {
+  return (
+    <React.Fragment>
+      <div className="w-3/4 mx-auto mt-4 md:mt-0">
+        <Image
+          src={billingImg3}
+          alt="Billing Image"
```
Comment (UCT-IT) [inline src/app/pages/billingAndPayroll/customizableInvoices/RightComponent.tsx]: alt should always come from locales

File: src/components/common/button/CommonButton.tsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ const CommonButton = ({ text, href, hover, color }: buttonProps) => {
   return (
     <Link href={href}>
       <button
-        className={`btn transition-all py-4 px-10 rounded hover:bg-${hover} bg-${color} text-white`}
+        className={`btn transition-all py-4 px-10 rounded hover:${hover} ${color} text-white`}
```
Comment (UCT-IT) [inline src/components/common/button/CommonButton.tsx:15]: Does this break other places where common button was used?

**Review by UCT-IT — CHANGES_REQUESTED** [review]

File: src/components/common/button/CommonButton.tsx (hunk 2 above)
Comment (Rasheduluctit) [reply src/components/common/button/CommonButton.tsx:15]: yes sir. Common Button colors didn't apply before the change. Now it's alright

File: src/app/pages/billingAndPayroll/customizableInvoices/RightComponent.tsx (hunk 1 above)
Comment (Rasheduluctit) [reply src/app/pages/billingAndPayroll/customizableInvoices/RightComponent.tsx]: okay

**Pull request merged**